src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {lottery.getRandomBall(), lottery.getRandomBall(), lottery.getRandomBall()};
+
+        for (Ball ball: balls) {
+            System.out.println(ball.toString());
```

Comment:
There is no need to call toString() explicitly inside System.out.println() method because it'll be called implicitly